.gitpod.yml

```diff
@@ -10,10 +10,19 @@ tasks:
   - name: Run backend
     command: |
       gp sync-await setup
-      if [ ! -f custom/conf/app.ini ]
-      then
-        mkdir -p custom/conf/
-        echo -e "[server]\nROOT_URL=$(gp url 3000)/" > custom/conf/app.ini
+      mkdir -p custom/conf/
```

Comment:
Can move this to line 25 because if the file exists, the directory must as well.

Reply:
Yes https://github.com/go-gitea/gitea/pull/29162/commits/31a44796d569bef3e155c58ae95a280e27b60901